scripts/functions/detect_systems/lsb_release

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+__rvm_detect_system_from_lsb_release()
+{
+  local __detected=true
+  local __system_name="$( awk -F'=' '$1=="DISTRIB_ID"{print $2}' /etc/lsb-release | head -n 1 | tr '[A-Z]' '[a-z]' | tr -d \" )"
+
+  case $__system_name in
+    elementary*)    _system_name="Elementary";;
+    linuxmint*)     _system_name="Mint";;
+    manjarolinux*)  _system_name="Manjaro";;
+    neon*)          _system_name="Ubuntu";;
+    solus*)         _system_name="Solus";;
+    ubuntu*)        _system_name="Ubuntu";;
+    *)              __detected=false;;
```

Comment:
this could be `return 1` and then the rest of function can be only setting the version/arch